DataStructures/SinglyLinkedList.php

```diff
@@ -1,13 +1,12 @@
 <?php
+
 /**
  *  Singly Linked List
  */
-
 class SinglyLinkedList
 {
     public ?SinglyLinkedList $next = null;
     public $data;
-
```

Comment:
Did the linter pick this up, or did you just remove this line?  I'm not sure I like not having a line break between the property declarations and the method definition in this class.

Reply:
I don't know why that happened. I'll fix it now.